firefox-ios/Client/Frontend/Home/TopSites/DataManagement/ContileProvider.swift

```diff
@@ -102,9 +102,9 @@ class ContileProvider: ContileProviderInterface, URLCaching, FeatureFlaggable {
     }
 
     private var resourceEndpoint: URL? {
-        if featureFlags.isCoreFeatureEnabled(.useStagingContileAPI) {
-            return URL(string: ContileProvider.contileStagingResourceEndpoint, invalidCharacters: false)
-        }
+//        if featureFlags.isCoreFeatureEnabled(.useStagingContileAPI) {
+//            return URL(string: ContileProvider.contileStagingResourceEndpoint, invalidCharacters: false)
+//        }
```

Comment:
I know it's in draft, but just in case 😄

Reply:
it has been added back!